### ISSUE TITLE
loss-ce constantly on 1.0000

### ISSUE THREAD

**Issue by Esaada** [cypw/PyTorch-MFNet#15]
Hi , when training or finetuning, I'm always getting loss-ce=1.000, any suggestions?

![image](https://user-images.githubusercontent.com/23050230/52327479-f7f07200-29f4-11e9-8a12-d9bb3943fba7.png)

**Comment by lxgyChen** [comment]
same with me
